src/gen/primitives/string.rs

```diff
@@ -34,8 +33,7 @@ impl Renderable for StringCodeType {
                 }
 
                 static int allocationSize([String value = ""]) {
-                    // FIXME: doing this twice for every string is bad
```

Comment:
technically this fixme hasn't been sold, has it?!? ... but well, I am not a fan of using fixme and other flags in the code so I am okay with removing it.

Reply:
okay, but probably not too bad of an idea to bring it back